wasm/vm_num_test.go

```diff
@@ -1,3 +1,94 @@
 package wasm
 
-// fixme
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type NumTestSuite struct {
+	suite.Suite
+	vm *VirtualMachine
+}
+
+func (suite *NumTestSuite) SetupTest() {
+	suite.vm = &VirtualMachine{
+		OperandStack: NewVirtualMachineOperandStack(),
+	}
+}
+
+func (suite *NumTestSuite) Testi32eqz() {
```

Comment:
should we add the case where the expected value is 1?

Reply:
ok